src/auth_middleware.rs

```diff
@@ -256,6 +239,29 @@ impl AuthMiddleware {
 
         Ok(())
     }
+
+    /// Extract the session token string from the Authorization header.
```

Comment:
nit: Also this method allows to extract the auth token from get parameters right? Maybe we should mention it on this comment.